scripts/data/allowedChains.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
configuration file, needed to specify which data will be updated from nova-utils. If we need to add a new network or asset, we can add this data to allowedChains.json and run the script to update it